src/main/java/reposense/frontend/RepoSense.java

```diff
@@ -27,19 +24,6 @@ public static void main(String[] args) {
             RepoInfoFileGenerator.generateReposReport(configs, targetFile.toAbsolutePath().toString());
         } catch (IllegalArgumentException ex) {
             System.out.print(ex.getMessage());
-            showHelpMessage();
         }
     }
-
-    private static void showHelpMessage() {
-        System.out.println("usage: java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH\n"
-                + "   [-output OUTPUT_DIRECTORY]\n"
-                + "   [-since DD/MM/YYYY]\n"
-                + "   [-until DD/MM/YYYY]\n");
-        System.out.println("-config: Mandatory. The path to the CSV config file.");
-        System.out.println("-output: Optional. The path to the dashboard generated.\n"
-                + "   If not provided, it will be generated in the current directory.");
-        System.out.println("-since : Optional. start date of analysis. Format: dd/MM/yyyy");
-        System.out.println("-until : Optional. end date of analysis. Format: dd/MM/yyyy");
```

Comment:
How about using the old messages instead?

Reply:
Yes. Done.